src/hooks/progress/use-tutorial-progress-mutations/index.ts

```diff
@@ -31,22 +33,25 @@ const useTutorialProgressMutations = (): UseTutorialProgressMutationsResult => {
 
 	/**
 	 * When a mutation is successful, we set the associated query data.
-	 * TODO: once there are other queries, we'll likely want to invalidate those.
+	 * We also invalidate queries that might involve related data.
 	 */
 	const makeOnMutationSuccess = () => {
 		return (
 			data: ApiCollectionTutorialProgress,
 			mutationVariables: TutorialProgressMutationVariables
 		) => {
-			// Destructure the variables we need to update related queries
+			// Destructure the variables we need to update tutorial query data
 			const { tutorialId, collectionId } = mutationVariables
 			queryClient.setQueryData(
-				[TUTORIAL_PROGRESS_QUERY_ID, tutorialId, collectionId],
+				[TUTORIAL_PROGRESS_SINGLE_QUERY_ID, tutorialId, collectionId],
 				data
 			)
 
 			// Invalidate related collection progress
-			queryClient.invalidateQueries(['collectionProgress', collectionId])
+			queryClient.invalidateQueries([
+				COLLECTION_PROGRESS_SINGLE_QUERY_ID,
```

Comment:
🚧 I need to figure out the requests-without-token `400` issue, which I think is related to `invalidateQueries` here... but _might_ try to leave that for a separate PR (I'm not sure how spiky of an issue it is yet).

Reply:
Ready to resolve this? 🥳